code/game/atoms.dm

```diff
@@ -501,6 +501,7 @@
 	if(. & COMPONENT_NO_EXPOSE_REAGENTS)
 		return
 
+	SEND_SIGNAL(source, COMSIG_REAGENTS_EXPOSE_ATOM, reagents, methods, volume_modifier, show_message)
```

Comment:
```suggestion
	SEND_SIGNAL(source, COMSIG_ATOM_EXPOSE_REAGENTS, reagents, methods, volume_modifier, show_message)
```
Use the existing signal conventions.

Reply:
I think you're misinterpreting the signal. This is posting to the reagent holder, not the atom.

Reply:
Aha, you are correct.
On the other change, do you plan to do it?

Reply:
@Rohesie `COMSIG_ATOM_EXPOSE_REAGENTS` is already being posed 4 lines up...

Reply:
I do, at some point. My next two weeks are packed with cramming and finals respectively.